gravitee-am-gateway/gravitee-am-gateway-services/gravitee-am-gateway-services-sync/src/main/java/io/gravitee/am/gateway/services/sync/SyncManager.java

```diff
@@ -267,6 +264,8 @@ private void synchronizeDomain(Event event) {
             case DELETE:
                 securityDomainManager.undeploy(domainId);
                 break;
+            default:
```

Comment:
break in default?

Can we not use a new switch here?

Reply:
https://sonarcloud.io/organizations/gravitee-io/rules?open=java%3AS131&rule_key=java%3AS131